.github/workflows/tests.yml

```diff
@@ -0,0 +1,66 @@
+name: tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  tests:
+
+    runs-on: ubuntu-latest
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: yes
+          MYSQL_DATABASE: forge
+        ports:
+          - 33306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+      redis:
+        image: redis:5.0
+        ports:
+          - 6379:6379
+        options: --entrypoint redis-server
+    strategy:
+      fail-fast: true
+      matrix:
+        php: [7.2, 7.3, 7.4]
+        stability: [prefer-lowest, prefer-stable]
+
+    name: P${{ matrix.php }} - S${{ matrix.stability }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Cache dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/.composer/cache/files
+          key: dependencies-php-${{ matrix.php }}-composer-${{ hashFiles('composer.json') }}
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: dom, curl, libxml, mbstring, zip, pcntl, pdo, sqlite, pdo_sqlite, gd
+          coverage: none
+
+      - name: Setup Memcached
+        uses: niden/actions-memcached@v7
+
+      - name: Install dependencies
+        run: composer update --${{ matrix.stability }} --prefer-dist --no-interaction --no-suggest
+
+      - name: Start MySQL
+        run: sudo /etc/init.d/mysql start
```

Comment:
@driesvints I'm a bit late to the party 😅 but why start mysql here when it's actually defined as a service and `env.DB_PORT` below in fact references the services' port?

I.e. seems this isn't needed?

Reply:
@mfn because of https://github.blog/changelog/2020-02-21-github-actions-breaking-change-ubuntu-virtual-environments-will-no-longer-start-the-mysql-service-automatically/

Reply:
@driesvints thanks for the answer!

Though i was referring to the fact that the GHA are using the MySQL instance from the `service` definition and not the local one, i.e. this isn't necessary and if you remove it, tests still work => see https://github.com/laravel/framework/pull/32010

Reply:
yeah but isn't that because the above changes aren't active yet?

Reply:
Sorry I don't follow :-/

Reply:
Nvm. It says March 3 so this should be good. Thanks! :)